cmd/ufs/ufs.go

```diff
@@ -1,13 +1,21 @@
-// This is a ufs server.
+// This is a userspace server which exports a filesystem over 9p(2000).
+//
+// UFS exports a filesystem from the server running it via 9p over
+// a network connection.
+//
+// By default, it will export / over a TCP on port 5640 under the username
+// of "harvey".
+//
+// UFS is unrelated to the UFS filesystem from BSD.
```

Comment:
I don't think this comment is needed. ufs is just a name. And the name ufs is pretty old, I am not sure it started with bsd.